odxtools/database.py

```diff
@@ -23,7 +24,7 @@ def __init__(self,
                  odx_d_file_name: str = None,
                  enable_candela_workarounds: bool = True):
 
-        self._id_lookup: Dict[str, Any] = {}
+        self._odxlinks: Dict[str, Any] = {}
```

Comment:
Wrong key type

Reply:
yep, but also superfluous local variable at this place. removed. thanks for the catch...